### ISSUE TITLE
UI Design?

### ISSUE THREAD

**Issue by raphaelaleixo** [cynicaloptimist/longwave#22]
Hi, would you be interested in a design for this app? :)

**Comment by cynicaloptimist** [comment]
I'd be happy to look at a mockup. I get a lot of traffic through RocketCrab: https://rocketcrab.com/ so it would be good if a design meshed well with their frame.

**Comment by raphaelaleixo** [comment]
You can take a look on some of the games I did recently:
https://krimi.ludoratory.com/ and https://unmatched.ludoratory.com/

I also did a version for Codenames, but I was asked to take it down. It's still on my github, though. :)

**Comment by cynicaloptimist** [comment]
Nice designs! Ok, what did you have in mind?

**Comment by raphaelaleixo** [comment]
Cool! I will work on something and show to you. :)

**Comment by raphaelaleixo** [comment]
Starting something:

![image](https://user-images.githubusercontent.com/5769966/112483647-9cf27400-8d4f-11eb-9304-0d84a42fc2d5.png)

**Comment by tqj5047** [comment]
Yes! and please remove the react logo from index.html. It's a big react logo everytime I share this website -- kind of weird. Thanks! (Just a minor issue so I didnt want to start a new thread)

**Comment by cynicaloptimist** [comment]
Hey, sorry I dropped this. That is a really nice design- I'd likely accept PRs for this. Try to make individual PRs relatively granular even if you're planning more of an overhaul. Thanks @raphaelaleixo !
